packages/design-system/src/components/Accordion/Primitive/CollapsiblePanel.tsx

```diff
@@ -1,20 +1,21 @@
-import { forwardRef, ReactChild, Ref, useState, useEffect, HTMLAttributes } from 'react';
+import { forwardRef, HTMLAttributes, ReactChild, Ref, useEffect, useState } from 'react';
+
 import classNames from 'classnames';
 
-import { useId } from '../../../useId';
 import { DataAttributes } from '../../../types';
+import { useId } from '../../../useId';
 import { variants } from '../../Status/Primitive/StatusPrimitive';
-
 import CollapsiblePanelHeader from './CollapsiblePanelHeader';
 import { PanelHeaderAction } from './types';
+
 import styles from './CollapsiblePanel.module.scss';
 
 type CollapsiblePanelCommonPropsType = {
 	children: ReactChild;
 	managed?: boolean;
 	expanded?: boolean;
 	index?: number;
-	action?: PanelHeaderAction;
+	actions?: [PanelHeaderAction, PanelHeaderAction?];
```

Comment:
What do you think about allowing unlimited number of actions ?
```suggestion
	actions?: PanelHeaderAction[];
```

Reply:
I think it's already supported by `actions.map(` you did in CollapsiblePanelHeader

Reply:
I like the idea however i was just following what was required in the ticket... but I will update as you said 🙂